### ISSUE TITLE
Ensure designs meet WCAG contrast check

### ISSUE THREAD

**Issue by beaudrykock-moi** [sfbrigade/care-connect#456]
Ensure all designs meet WCAG contrast checks.

We can start with buttons and critical communication but ideally should be WCAG-passing throughout.

This is not a P0 task - can wait until post-v4 and v5.

**Comment by coilin** [comment]
link with https://github.com/sfbrigade/care-connect/issues/640
